modules/monitoring-configuring-a-persistent-volume-claim.adoc

```diff
@@ -89,3 +89,5 @@ data:
               storage: *40Gi*
 ----
 
+. Finish editing the ConfigMap. The pods affected by the new configuration will now be restarted automatically and the new storage configuration will be applied.
```

Comment:
Thanks! Applied this, apart from the "pods" versus "Pods". As far as I can see, we have "pods" everywhere.

Reply:
Yeah, the guidelines [1] say we should be using "Pod" uppercase, but honestly there is a lot of inconsistency since not everyone has been following that. So it's fine for now.

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#pod